src/qt/lookupspdialog.cpp

```diff
@@ -59,6 +59,35 @@ LookupSPDialog::LookupSPDialog(QWidget *parent) :
     ui->topFrame->setVisible(false);
     ui->leftFrame->setVisible(false);
     ui->rightFrame->setVisible(false);
+    ui->denominationTable->setVisible(false);
+
+    // denominations
+    ui->denominationTable->setColumnCount(2);
+    ui->denominationTable->setHorizontalHeaderItem(0, new QTableWidgetItem("Denomination"));
+    ui->denominationTable->setHorizontalHeaderItem(1, new QTableWidgetItem("Value"));
+    borrowedColumnResizingFixer = new GUIUtil::TableViewLastColumnResizingFixer(ui->denominationTable, 50, 150, this);
+    // note neither resizetocontents or stretch allow user to adjust - go interactive then manually set widths
+    #if QT_VERSION < 0x050000
+       ui->denominationTable->horizontalHeader()->setResizeMode(0, QHeaderView::Stretch);
+       ui->denominationTable->horizontalHeader()->setResizeMode(1, QHeaderView::Stretch);
+    #else
+       ui->denominationTable->horizontalHeader()->setSectionResizeMode(0, QHeaderView::ResizeToContents);
+       ui->denominationTable->horizontalHeader()->setSectionResizeMode(1, QHeaderView::Stretch);
+    #endif
+    ui->denominationTable->setAlternatingRowColors(true);
+
+    // initial resizing
+    ui->denominationTable->resizeColumnToContents(0);
+    // borrowedColumnResizingFixer->stretchColumnWidth(0);
+    ui->denominationTable->verticalHeader()->setVisible(false);
+    ui->denominationTable->setHorizontalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
+    ui->denominationTable->setSelectionBehavior(QAbstractItemView::SelectRows);
+    ui->denominationTable->setSelectionMode(QAbstractItemView::SingleSelection);
+    ui->denominationTable->setVerticalScrollBarPolicy(Qt::ScrollBarAlwaysOn);
+    ui->denominationTable->setTabKeyNavigation(false);
+    ui->denominationTable->setContextMenuPolicy(Qt::CustomContextMenu);
```

Comment:
This required for custom context menu (right click menu). I don't think we need this.

Reply:
nice catch